pylutron_caseta/smartbridge.py

```diff
@@ -989,6 +1023,70 @@ def _process_occupancy_group(self, occgroup):
             name=f"{self.areas[occgroup_area_id]['name']} Occupancy",
         )
 
+    async def _load_ra3_occupancy_groups(self):
+        """Load the devices from the bridge and filter for occupancy sensors."""
+        _LOG.debug("Finding occupancy sensors from bridge")
+        occdevice_json = await self._request(
+            "ReadRequest", "/device?where=IsThisDevice:false"
```

Comment:
Is there a way to query just the occupancy sensors? If there is only one kind of occupancy sensor we could filter by type. The test data doesn't give many hints.

Reply:
Unfortunately, I have not been able to find a way to query just the occupancy sensors.  This is the only way I've found to discover them at all.